src/components/Table/components/EditColumnsButton.tsx

```diff
@@ -76,22 +74,27 @@ export function EditColumnsButton<R extends Kinded>(props: EditColumnsButtonProp
     <OverlayTrigger {...props} menuTriggerProps={menuTriggerProps} state={state} buttonRef={buttonRef} {...tid}>
       <div
         css={{
-          ...Css.bgWhite.py5.px3.maxwPx(380).bshBasic.$,
+          ...Css.dg.gtc("1fr auto").gap2.bgWhite.p2.maxwPx(326).$,
           "&:hover": Css.bshHover.$,
         }}
       >
-        <div css={Css.gray500.xsSb.mb1.ttu.$}>{title || "Select columns to show"}</div>
-        <CheckboxGroup
-          label={title || "Select columns to show"}
-          onChange={(values) => setSelectedValues(values)}
-          values={selectedValues}
-          options={options}
-          columns={2}
-          labelStyle="hidden"
-        />
-        <div css={Css.mt1.$}>
-          <Button variant={"tertiary"} label={"Clear selections"} onClick={() => setSelectedValues([])} />
-        </div>
+        {options.map((option) => (
+          <Fragment key={option.value}>
+            <div css={Css.sm.truncate.pr1.$}>{option.label}</div>
```

Comment:
Totally fine to keep the `truncate`, but I kinda wonder if "the label would look bad untruncated", we should leave as-is so callers know/see to "pick better labels" :-) 🤷